plugins/inventory/inventory.py

```diff
@@ -0,0 +1,162 @@
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2022, XLAB Steampunk <steampunk@xlab.si>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+DOCUMENTATION = r"""
+name: inventory
```

Comment:
Rename the inventory plugin to `terraform_provider` please, to signal that this is meant to be used with terraform-provider-ansible.

Reply:
renamed